File: modules/wem-api/src/main/java/com/enonic/wem/query/expr/FieldExpr.java
package com.enonic.wem.query.expr;

public final class FieldExpr
    implements Expression
{
    public final String name;

    public FieldExpr( final String name )
    {
        this.name = name;
    }

    public String getName()
    {
        return this.name;
    }

    @Override
    public String toString()
    {
        return this.name;
    }
}

File: modules/admin/admin-ui/src/main/resources/web/admin/common/js/query/expr/FieldExpr.ts
module api.query.expr {

    export class FieldExpr implements Expression {

        private name: string;

        constructor(name: string) {
            this.name = name;
        }

        getName(): string {
            return this.name;
        }

        toString() {
            return this.name;
        }
    }
}
